packages/cq-cert-runner/src/index.ts

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env node
+import { IHEGateway, APIMode } from "@metriport/ihe-gateway-sdk";
+import { Command } from "commander";
+import { patients } from "./payloads";
+// import * as dotenv from "dotenv";
+
+function metriportBanner(): string {
+  return `
+            ,▄,
+          ▄▓███▌
+      ▄▀╙   ▀▓▀    ²▄
+    ▄└               ╙▌
+  ,▀                   ╨▄
+  ▌                     ║
+                         ▌
+                         ▌
+,▓██▄                 ╔███▄
+╙███▌                 ▀███▀
+    ▀▄
+      ▀╗▄         ,▄
+         '╙▀▀▀▀▀╙''
+
+        Metriport Inc.
+
+    CQ Cert Runner
```

Comment:
ditto: CQ > Carequality

Reply:
hypernit: more spaces as well to center it